backend/core/database.py

```diff
@@ -1226,13 +1245,34 @@ def handle_hijack_multiple_action(self, message):
                                         query, (datetime.datetime.now(), hijack_key)
                                     )
                             elif delete_action:
+                                redis_hijack_key = redis_key(raw["prefix"], raw["hijack_as"], raw["type"])
```

Comment:
this doesn't work right now as raw only includes a field called keys that has the selected hijack keys.
@pgigis if we can add additional information so it can be something like a list that inlcudes, key, prefix, as and type then this can work.

Pretty much we want the same functionality as we have it for the single deletion.

Reply:
@pgigis what is the status on this? Could you please implement this so that the PR is not left hanging?

Reply:
@slowr could you re-add this comment in the proper place where you need JS support because due to the current changes it became outdated?